src/Symfony/Component/DependencyInjection/ServiceCollectionInterface.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\DependencyInjection;
+
+use Psr\Container\ContainerInterface as PsrContainerInterface;
+
+/**
+ * @author Kevin Bond <kevinbond@gmail.com>
+ *
+ * @template-covariant T of mixed
+ *
+ * @extends PsrContainerInterface<T>
```

Comment:
This is wrong to me given that PsrContainerInterface is not defined as a generic interface.